index.rst

```diff
@@ -70,292 +70,140 @@ In gross, the conceptual steps are:
-
-Note that the difference in *git tag* name convention between first and third
-parties is automatically handled by the ``--external-team`` flag.
-
-.. code-block:: bash
-
-   # create git "release candidate" tag from manifest ID
-
-   github-tag-version \
-     --debug \
-     --token **** \
-     --user sqreadmin \
-     --email sqre-admin@lists.lsst.org \
-     --org lsst \
-     --allow-team 'Data Management' \
-     --allow-team 'DM Externals' \
-     --external-team 'DM Externals' \
-     --deny-team 'DM Auxilliaries' \
-     --manifest-only \
-     --manifest b2999 \
-     v42.0.0.rc1
-
-.. code-block:: bash
-
-    # create git on aux repos
-    # previous weekly tag.
-
-    github-tag-teams \
-      --debug \
-      --token **** \
-      --user sqreadmin \
-      --email sqre-admin@lists.lsst.org \
-      --org lsst \
-      --allow-team 'DM Auxilliaries' \
-      --deny-team 'DM Externals' \
-      --ignore-existing-tag \
-      --tag v42.0.0.rc1
-
-**XXX this is currently broken in that the git tag will be placed at the
-current HEAD of the default branch instead of at the same location as the**
-
-Build and Publish eups ``eupspkg`` packages + eups tag
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-https://ci.lsst.codes/blue/organizations/jenkins/release%2Frun-rebuild/activity
-
-The resulting manifest ID needs to be retrieve to use as input for subsequent
-jobs.
+1st Release Candidate
+---------------------
 
-.. code-block::
+Build and Publish
+^^^^^^^^^^^^^^^^^
 
-    REFS: v42.0.0.rc1
-    PRODUCTS: lsst_distrib
-    BUILD_DOCS: true
+Run the jenkins `release/official-release <official-release>`_ job.  The source
+git refs should be only the tag of the "seed" weekly release.  The release tag
+**must** start with a ``v`` and end with ``.rc1``.
 
-https://ci.lsst.codes/blue/organizations/jenkins/release%2Frun-publish/activity
+See git-tags_ for details on the formatting of git tags.
 
-.. code-block::
+Example:
 
-    PRODUCTS: lsst_distrib
-    EUPSPKG_SOURCE: git
-    TAG: v42_0_0_rc1
-    BUILD_ID: bXXXX
+.. code-block:: text
 
-Build and Publish eups ``tarball`` packages
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   SOURCE_GIT_REFS: w.1999.42
+   RELEASE_GIT_TAG: v42.0.0.rc1
+   O_LATEST: false
 
-Note that the jenkins ``release/official-release`` job does not trigger
-``release/tarball-matrix`` and triggers ``release/tarball`` build(s) directly
-so as to have more explicit control over the parameters.
-
-https://ci.lsst.codes/blue/organizations/jenkins/release%2Ftarball-matrix/activity
+Announce rc1
+^^^^^^^^^^^^
 
-.. code-block::
+Update the release status `community.lsst.org <clo>`_ post to to announce the
+availability of ``rc1``.
 
-    PRODUCTS: lsst_distrib
-    EUPS_TAG: v42_0_0_rc1
-    SMOKE: true
-    RUN_SCONS_CHECK: true
-    PUBLISH: true
 
-Build and Publish ``scipipe`` docker image
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+2nd+ Release Candidate(s)
+-------------------------
```

Comment:
A blurb about test building an rcX candidate with `stack-os-matrix` prior to running `release/official-release` is needed here.

Reply:
This should probably be done by who is in charge of merging the changes into the release branch. 
Maybe this should be automated: as soon as some changes are merged in the release branch, the stack-os-matrix job should be run in order to ensure that the changes are fine.